src/tests/bulkConcepts/actions/bulkConcept.test.js

```diff
@@ -233,6 +237,65 @@ describe('Test suite for addBulkConcepts async actions', () => {
     return store.dispatch(setPreviousPage())
       .then(() => expect(store.getActions()).toEqual(returnedAction));
   });
+  it('should notify user when it fails to add a concept', () => {
+    const notifyMock = jest.fn();
+    notify.show = notifyMock;
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        response: [{ concepts, ...{ added: false } }],
+      });
+    });
+
+    const store = mockStore({});
+    const params = { type: 'user', typeName: 'emasys', collectionName: 'dev jam' };
+    return store.dispatch(addConcept(params, [], 'lob dev')).then(() => {
+      expect(notifyMock).toHaveBeenCalledTimes(1);
+      expect(notifyMock).toHaveBeenCalledWith('Failed to add lob dev. Please Retry', 'error', 3000);
+    });
+  });
+});
+
+describe('recursivelyFetchConceptMappings', () => {
+  const conceptCode = 100;
+  let fetchFromPublicSourcesMock;
+
+  beforeEach(() => {
+    fetchFromPublicSourcesMock = jest.fn()
+      .mockReturnValueOnce({ data: [mappings[0]] })
+      .mockReturnValueOnce({ data: [mappings[1]] });
+    api.mappings.fetchFromPublicSources = fetchFromPublicSourcesMock;
+  });
+
+  it('returns a single level of mappings when called with 0 levelsToCheck', async () => {
```

Comment:
The description does not follow the convention of the rest.

Reply:
I have adjusted the description to follow conventions